opba-protocols/xs2a-protocol/src/main/resources/processes/xs2a/authorization/payment/xs2a-authorize-payment.bpmn20.xml

```diff
@@ -23,7 +23,7 @@
     <exclusiveGateway id="isMultipleScaAvailable" name="Multiple SCA available?" default="oneScaAvailable"/>
     <exclusiveGateway id="isScaFinalized" name="SCA finalized?" default="moreScaChallenges"/>
     <serviceTask id="persistPaymentAndContext" name="Persist singlePaymentBody and its context for future service session invocation" flowable:async="true" flowable:exclusive="false" flowable:delegateExpression="${xs2aPisPersistPaymentAndContext}"/>
-    <sequenceFlow id="oneScaAvailable" name="One SCA" sourceRef="isMultipleScaAvailable" targetRef="embeddedScaChallenge"/>
+    <sequenceFlow id="oneScaAvailable" name="One SCA" sourceRef="isMultipleScaAvailable" targetRef="isDecoupledScaSelected"/>
```

Comment:
Please apply the same comment fixes as in `xs2a-authorize-consent.bpmn20.xml `

Reply:
Done